src/Reflection/ReflectionClass.php

```diff
@@ -1097,7 +1089,7 @@ public function getStaticPropertyValue($propertyName)
      * @param mixed $value
      * @return void
      */
-    public function setStaticPropertyValue($propertyName, $value)
+    public function setStaticPropertyValue(string $propertyName, $value) : void
     {
         if (!class_exists($this->getName(), false)) {
```

Comment:
Yup, but at this point the user probably wants that to happen anyway. Maybe we should store the value when we hijack the autoloading, and set it when the actual autoloading happens? Better than throwing an exception. This needs further discussion though.